apps/src/blockly/addons/cdoSpritePointer.js

```diff
@@ -0,0 +1,112 @@
+// Helper functions for pointer block logic. In our pointer blocks (a.k.a.)
```

Comment:
At some point, I think we should try to re-organize how files of helpers like this one relate to other `addons/cdo*.js` files which tend to primarily export some custom class that is used/registered directly. I did some spot-checking and determined that we already have inconsistencies here - most of which are my fault. :)
That is to say, I think this file is okay for now, but we might find we want a different home for random block helpers down the road.

Reply:
Agreed, I think we could refactor `addons` to be multiple folders. That feels out of scope for this though.

Reply:
Definitely out of scope!